assembl/static2/js/app/routes.jsx

```diff
@@ -24,10 +28,6 @@ export default (
       <Route path=":slug/community" component={Community} />
       <Route path=":slug/terms" component={Terms} />
     </Route>
-    <Route path="/v2/">
-      <Route path=":slug/profile" component={Profile} />
-      <Route path=":slug/styleguide" component={Styleguide} />
-    </Route>
```

Comment:
All components below App need a discussion ID. May be I have to rename "App" component because this is confusing.